pkg/installer/install.go

```diff
@@ -39,17 +40,19 @@ type packageManager struct {
 	repositories *repository.Repositories
 	configsDir   string
 	installLock  sync.Mutex
+	tmpDirPath   string
 }
 
 func newPackageManager(repositories *repository.Repositories) *packageManager {
 	return &packageManager{
 		repositories: repositories,
 		configsDir:   defaultConfigsDir,
+		tmpDirPath:   filepath.Join(setup.InstallPath, "run"),
```

Comment:
I guess it probably doesn't happen (at all?) but if /opt/datadog-installer and /opt/datadog-packages are also on different filesystems, that wouldn't work either.